base_tier_validation_formula/models/tier_review.py

```diff
@@ -10,8 +10,11 @@ class TierReview(models.Model):
     _inherit = "tier.review"
 
     python_reviewer_ids = fields.Many2many(
+        "res.users",
+        "tier_review_python_reviewer_rel",
+        "tier_review_id",
+        "user_id",
```

Comment:
can you be explicit with the attributes?

Reply:
done